Jropp/jason-ropp-js-iobp
# Onboarding Proejct: Mini-Project, PWA Perfomance

## What It Does

Implements PWA changes into main project.

To upgrade project to operate near PWA standards.

- I included a manifest that includes styles for the splash page and icons.
- I added a caching web worker.
- I added a fallback four win there is no JavaScript present.

## How To Test

Open dev tools and you should see logged messages from work box showing the caching of the load files.

Writing a lighthouse audit on the page shop now score 88 under PWA.

## Notes/Caveats

## Checklist

Under penalty of public shaming, I avow that I:


Browsers tested:

- [ ] ~Internet Exploder~

## Dependencies

- [ ] Nada
